Bottom: 4448ffa41ba65496f4d013232ccf74c6ff1cec5e
Top:    02ad2e09f95b7ad807227f6b38e36cc5a5039fc8
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-03-27 20:11:59 +0100

Refresh of remove-blocktoo-argument-from--0

---

diff --git a/src/crosshair.c b/src/crosshair.c
index 983ac6b..792d9c2 100644
--- a/src/crosshair.c
+++ b/src/crosshair.c
@@ -671,7 +671,8 @@ DrawAttached (void)
     }
 
   /* an attached box does not depend on a special mode */
-  if (Crosshair.AttachedBox.State == STATE_SECOND)
+  if (Crosshair.AttachedBox.State == STATE_SECOND ||
+      Crosshair.AttachedBox.State == STATE_THIRD)
     {
       LocationType x1, y1, x2, y2;
